fairlearn/metrics/group_metric_result.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+class GroupMetricResult:
+    def __init__(self):
```

Comment:
Is the empty constructor arg list intended? Wouldn't it be easier to just provide everything in the constructor? Maybe I've missed the reason.

Reply:
Yes. I was having the constructor make sure a bunch of fields exist

Reply:
Oh great! Let me know if I'm reviewing this too early. I can hold off :-)

Reply:
Sorry, the "yes" was for "empty argument list intended"